src/mods/depositAll/index.js

```diff
@@ -0,0 +1,35 @@
+import { makeElement } from '../../utils/misc';
+import { getWindow } from '../../utils/game';
+import { deposit } from './helper';
+
+function addDepositAllButton() {
+	const $stash = getWindow('Stash');
+	// If stash is closed or deposit all button is already added, we dont need to do anything
+	if (!$stash || $stash.querySelector('.js-deposit-all')) {
+		return;
+	}
+
+	// Create deposit all button and add it to stash
+	const depositTargetBtn = document.querySelector('.slot .grey.gold');
```

Comment:
Nitpick, but: Code base has been prefixing all DOM elements, i.e. objects grabbed via `.querySelector()`, with a `$`. A habit I picked up a long while ago, probably from jQuery or something.
Helps identify when you're dealing with DOM elements in your JS. I find it makes the code a little easier to read.

Reply:
Oh sweet, I had wondered about the $varNames but forgot to ask, updated!